### ISSUE TITLE
General Issues about button disappearing in new file viewer version of Github 

### ISSUE THREAD

**Issue by ericvd-ucb** [yuvipanda/nbgitpuller-link-generator-webextension#2]
For me the experience was that button disappeared, when I was previewing a new version of Github

**Comment by balajialg** [comment]
Yes +100 to @ericvd-ucb's comment as this new and forced global navigation update in Github is proving to be a huge pain point for using this nbgitpuller extension @yuvipanda. 

**Snapshot 1:** 
![image](https://github.com/yuvipanda/nbgitpuller-link-generator-webextension/assets/2306166/3907076a-0e5d-4989-ad22-793c83c0b6de)

**Snapshot 2:**
![image](https://github.com/yuvipanda/nbgitpuller-link-generator-webextension/assets/2306166/285d1665-c117-46e8-9543-13090bd151c3)

The nbgitpuller button disappears when navigated to the file browser as seen in the snapshot 2 making this plugin unusable across Firefox and Chrome. We need to fix this before the start of summer classes or else we will be going back by 2 years to [generate links in the old-fashioned way](https://nbgitpuller.readthedocs.io/en/latest/link.html). I am not sure what the fix here is but your inputs here will be a huge value add.

I gave this feedback in the comments section of the [Global navigation beta feedback page](https://github.com/orgs/community/discussions/52083#discussioncomment-5933193) but not sure whether anyone would care.

**Comment by pamelafox** [comment]
Sent fix in #5

**Comment by yuvipanda** [comment]
Thanks a lot, @pamelafox! I've tagged a v2, and updated the firefox extension - it is awaiting review. I'll do the same for Chrome

**Comment by yuvipanda** [comment]
I've submitted the extension to Chrome store for review as well!

**Comment by balajialg** [comment]
Wow, that's awesome @pamelafox and @yuvipanda! Super excited about this fix :)

**Comment by balajialg** [comment]
Firefox extension is updated (Chrome extension is not yet updated) and I can test this fix. Thanks @pamelafox for unblocking us with this fix.

![image](https://github.com/yuvipanda/nbgitpuller-link-generator-webextension/assets/2306166/45543ba2-7777-4ad7-b952-89ecc0fc2f21)

**Comment by balajialg** [comment]
@pamelafox Your point about removing this extension's dependency on Github makes complete sense. I am running into issues such as this on a regular basis wrt specific repositories.
![image](https://github.com/yuvipanda/nbgitpuller-link-generator-webextension/assets/2306166/2fac348d-a7a3-4632-8aec-f64bdcccb951)

**Comment by balajialg** [comment]
@pamelafox @ericvd-ucb Seems like the nbgitpuller plugin is no longer visible when navigated to the specific ipynb file.

![image](https://github.com/yuvipanda/nbgitpuller-link-generator-webextension/assets/2306166/55b6a8c4-214a-4802-93df-4ffb1074a790)

![image](https://github.com/yuvipanda/nbgitpuller-link-generator-webextension/assets/2306166/f1f28ecc-53e6-4ad1-a3d2-4533173b6df6)
